index.bs

```diff
@@ -2760,7 +2760,8 @@ The <dfn export for=commands>browsingContext.create</dfn> command creates a new
       BrowsingContextCreateType = "tab" / "window"
 
       BrowsingContextCreateParameters = {
-        type: BrowsingContextCreateType
+        type: BrowsingContextCreateType,
+        ?context: BrowsingContext
```

Comment:
Maybe like this to avoid confusion? E.g. users can expect the given context to be replaced with the new one.
```suggestion
        ?referenceContext: BrowsingContext
```